src/hooks/useAllPositions.ts

```diff
@@ -37,88 +38,89 @@ export const useAllPositions = (
   const { honeyUser, honeyReserves, honeyMarket } = useMarket(connection, wallet, honeyId, honeyMarketId);
   const { marketReserveInfo } = useHoney();
 
-  const fetchPositions = async () => {
```

Comment:
@yuriagile please add this function back so we don't need to change the frontend before merging the UX update
we can migrate afterwards.